staging/src/k8s.io/cri-api/go.mod

```diff
@@ -7,17 +7,15 @@ module k8s.io/cri-api
 go 1.12
 
 require (
-	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/emicklei/go-restful v0.0.0-20170410110728-ff4f55a20633
 	github.com/gogo/protobuf v1.1.1
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b // indirect
-	github.com/golang/protobuf v1.2.0 // indirect
-	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/stretchr/testify v1.2.2
 	golang.org/x/net v0.0.0-20190206173232-65e2d4e15006
-	golang.org/x/sync v0.0.0-20181108010431-42b317875d0f // indirect
-	golang.org/x/text v0.3.0 // indirect
-	google.golang.org/genproto v0.0.0-20170731182057-09f6ed296fc6 // indirect
 	google.golang.org/grpc v1.13.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/apiserver v0.0.0 // indirect
+	k8s.io/client-go v0.0.0
+	k8s.io/kubernetes v1.14.0
```

Comment:
staging repos shouldn't be using k/k

Reply:
https://github.com/kubernetes/kubernetes/blob/97011635f727307db25566a0459a51c6e1663836/staging/src/k8s.io/cri-api/pkg/streaming/server.go#L38

Reply:
also:
https://github.com/kubernetes/kubernetes/blob/97011635f727307db25566a0459a51c6e1663836/staging/src/k8s.io/cri-api/pkg/streaming/server.go#L39